POEApi.Model/StackInfo.cs

```diff
@@ -1,8 +1,13 @@
-﻿using System.Diagnostics;
+﻿using System;
+using System.Diagnostics;
 
 namespace POEApi.Model
 {
+    //Todo: Stackable items now have it in the base of the message - rather than the properties. 
```

Comment:
Is this a recent change in the JSON?

Reply:
I don't know how recent but the bin file that has been passed over by yourself definitely has these properties.

Reply:
Acknowledged.  I must've missed it then.  Good to know it's there in a more easily parsable format!